tests/e2e/ekscluster/furyctl_apply.expect

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/expect -f
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+
+# Increase timeout as needed:
+set timeout 3600
+puts "Current working directory: $env(PWD)"
+if {[llength $argv] == 0} {
+    puts "Usage: furyctl_apply.expect /path/to/furyctl.yaml"
+    exit 1
+}
+
+if {[llength $argv] >= 2} { 
+    set outdir [lindex $argv 1] 
+    puts "Using specified output directory: $outdir" 
+} else { 
+    set outdir $env(PWD) 
+    puts "Using current directory as output directory: $outdir" 
+}
+
+set configFilePath [lindex $argv 0]
+# Verify that necessary AWS environment variables exist.
+if { ![info exists env(AWS_ACCESS_KEY_ID)] || \
+     ![info exists env(AWS_SECRET_ACCESS_KEY)] || \
+     ![info exists env(AWS_REGION)] } {
+    puts "One or more AWS environment variables (AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION) are not set."
+    exit 1
+}
+
+
+# Start furyctl apply.
+spawn furyctl apply --outdir $outdir -D --config $configFilePath
```

Comment:
we should set the `-d` (`--disable-analytics`) flag too, it does not make sense to have it enabled in CI

also, it is a good practice to use the long-form of the flags in scripts to improve readability:
```suggestion
spawn furyctl apply --outdir $outdir --debug --disable-analytics --config $configFilePath
```